_settings/mappings/views.json

```diff
@@ -28,38 +37,24 @@
     "comment_status": {
       "type": "string"
     },
-    "custom_fields": {
-      "properties": {
-        "popular_posts": {
-          "type": "string"
-        },
-        "related_links_0_link": {
-          "type": "string"
-        },
-        "related_links_1_link": {
-          "type": "string"
-        },
-        "related_links_2_link": {
-          "type": "string"
-        },
-        "related_links_3_link": {
-          "type": "string"
-        },
-        "related_links_4_link": {
-          "type": "string"
-        },
-        "related_post": {
-          "type": "string"
-        }
-      }
-    },
     "date": {
-      "type": "string"
```

Comment:
Ah, right I forgot you had mentioned beta had issues on that. All the dates are like that. Should they roll back to strings and then be converted in the processor like it is currently?